libraries/type-length-value/src/state.rs

```diff
@@ -14,14 +14,10 @@ use {
 };
 
 /// Get the current TlvIndices from the current spot
-const fn get_indices_unchecked(type_start: usize) -> TlvIndices {
+const fn get_indices_unchecked(type_start: usize) -> (usize, usize, usize) {
```

Comment:
I had this written and then removed it, because to me it didn't really make sense. `get_indices_unchecked` is just adding to `type_start` to get `length_start` and `value_start`, and doesn't know or care what the discriminator is, so it can't possibly determine which repetition number it's on.

It would have to be just a literal pass-through value, which I can do, but it felt weird.

Reply:
it's an internal function, so I'm fine with just doing a pass-through for that repetition considering it'll avoid a big tuple in favor of a struct, and also help readability in the other functions. do you have a particular concern with it?

Reply:
Nope, just seemed weird that's all! 
I just added it, and it cleans up (and makes sense with) `get_indices` since it returns the same indices, but it looks a little weird with `get_discriminators_and_end_index`. However, since that function doesn't return any indices or repetition number, I think it's all good.